pkg/option/flags.go

```diff
@@ -252,6 +254,7 @@ func ReadAndSetFlags() error {
 
 	Config.CompatibilitySyscall64SizeType = viper.GetBool(KeyCompatibilitySyscall64SizeType)
 
+	Config.ExecveMapEntries = viper.GetString(KeyExecveMapEntries)
```

Comment:
proposition: maybe you could call `GetExecveMapEntries` so that you do the flag parsing here and you have a number in Config.ExecveMapEntries?